Tasks/DownloadPackageV0/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "downloadpackage",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
shouldn't the azure-devops-node-api version be updated on the DownloadPackageV0 task as well?